firestore/FirestoreSwiftUIExample/Views/PriceView.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  PriceView.swift
+//  FirestoreSwiftUIExample
+//
+
+import SwiftUI
+
+struct PriceView: View {
+  var price: Int
+
+  var body: some View {
+    Text(priceString(from: price))
+      .font(.footnote)
+      .foregroundColor(Color.gray)
+      .multilineTextAlignment(.trailing)
+  }
+
+  func priceString(from price: Int) -> String {
+    let priceText: String
```

Comment:
In this particular case, you might be able to use `String(repeating: "$", count: price)`

https://developer.apple.com/documentation/swift/string/2427723-init

Reply:
Cool! Will do